pex/bin/pex.py

```diff
@@ -876,7 +876,11 @@ def do_main(
             compress=options.compress,
         )
         if options.seed != Seed.NONE:
-            seed_info = seed_cache(options, pex, verbose=options.seed == Seed.VERBOSE)
+            seed_info = seed_cache(
```

Comment:
Tests exposed the fact seeding operated against the (loose) tmp PEX being build up and not the final built PEX. This fixes that - there was never a reason to seed from anything but the final PEX.